src/app/dev/DevToys.Tools/Tools/Text/ListCompare/ListCompareCommandLineTool.cs

```diff
@@ -0,0 +1,113 @@
+﻿using DevToys.Tools.Helpers;
+using DevToys.Tools.Models;
+using Microsoft.Extensions.Logging;
+using OneOf;
+
+namespace DevToys.Tools.Tools.Text.ListCompare;
+
+[Export(typeof(ICommandLineTool))]
+[Name("ListCompare")]
+[CommandName(
+    Name = "listcompare",
+    Alias = "lcompare",
+    ResourceManagerBaseName = "DevToys.Tools.Tools.Text.ListCompare.ListCompare",
+    DescriptionResourceName = nameof(ListCompare.Description))]
+internal sealed class ListCompareCommandLineTool : ICommandLineTool
+{
+#pragma warning disable IDE0044 // Add readonly modifier
+    [Import]
+    private IFileStorage _fileStorage = null!;
+#pragma warning restore IDE0044 // Add readonly modifier
+
+    [CommandLineOption(
+        Name = "filea",
+        Alias = "a",
+        IsRequired = true,
+        DescriptionResourceName = nameof(ListCompare.PathListA))]
+    private OneOf<FileInfo, string>? FileA { get; set; }
+
+    [CommandLineOption(
+        Name = "fileb",
+        Alias = "b",
+        IsRequired = true,
+        DescriptionResourceName = nameof(ListCompare.PathListB))]
+    private OneOf<FileInfo, string>? FileB { get; set; }
+
+    [CommandLineOption(
+        Name = "casesensitive",
+        Alias = "cs",
+        IsRequired = true,
```

Comment:
I would like to suggest to make this parameter (and OutputFile and ComparisonMode) optional. You can give a default value to the property, which will be displayed in `--help` as the default one.

Reply:
All right